Strutured the Engagement Site

NadiSanli opened the pull request:
Structured the Site and added Boxes, Pictures and Text

SapozhnikovD commented:
The indentation style we're using is 4 spaces per indentation level for HTML, CSS and PHP, with no exceptions for <head> and <body> (so they are 4 spaces to the right of <html>, not on the same level). If you are using an automatic formatter, please configure it to follow that style and format the files again.

SapozhnikovD commented on src/engagement.html:
```diff
@@ -1,78 +1,128 @@
+			<img src="img/suelogo5.jpg" id="logo" alt="SUE Logo">
+		</header>
+		<main>
+			<div class=background_engage>
```
1. It's better to use quotes for attributes even if you don't need to
2. Classes should be named in lowerCamelCase

So this should be `<div class="backgroundEngage">`. Same for all below

NadiSanli commented:
Hi Daniil,

In my opinion i am finished with my Code.
Please take a look at it and maybe give me some improvement tips.
I've fixed the indent Problem and i've added some comments on my Code.
